src/pf_types.h

```diff
@@ -2728,14 +2728,20 @@ typedef struct pf_lldp_port
    bool is_peer_info_received;
 } pf_lldp_port_t;
 
+typedef struct pf_netif
+{
+   char name[PNET_INTERFACE_NAME_MAX_SIZE]; /**< Terminated string */
+   pnet_ethaddr_t mac;
+   pnal_eth_handle_t * eth_handle;
```

Comment:
```suggestion
   pnal_eth_handle_t * handle;
```
Note that at point-of-use, this will read `netif.handle`.

Reply:
Done